crates/api/src/local_user/reset_password.rs

```diff
@@ -31,5 +34,5 @@ pub async fn reset_password(
 
   // Email the pure token to the user.
   send_password_reset_email(&local_user_view, &mut context.pool(), context.settings()).await?;
-  Ok(Json(PasswordResetResponse {}))
+  Ok(HttpResponse::Ok().finish())
```

Comment:
Use the `SuccessResponse` you just created, that's much easier to use for front ends.

Reply:
Have to wait for https://github.com/LemmyNet/lemmy/pull/3976 then.